pages/[lng]/article/[slug].tsx

```diff
@@ -1,7 +1,12 @@
 /* library package */
 import { FC, useState } from 'react'
 import { GetServerSideProps, InferGetServerSidePropsType } from 'next'
-import { Article, ArticleCategories, useI18n } from '@sirclo/nexus'
+import {
+  Article,
+  ArticleCategories,
+  useI18n,
+  useAuthToken,
+} from '@sirclo/nexus';
```

Comment:
```suggestion
} from '@sirclo/nexus'
```

titik koma-nya dihapus pacc biar seragam
mungkin kalo ada yang di file lain juga sama dibegitukan